framework/src/bundle/CoreBundleContext.h

```diff
@@ -187,6 +199,11 @@ namespace cppmicroservices
 
         void Uninit1();
 
+        WriteLock SetFrameworkStopped();
+
+        WriteLock SetFrameworkStarted();
```

Comment:
these are better names. I think there is room for improvement on the naming. Also, as a developer who may have to add calls to these methods in the cppms code base, I don't know how to use these methods. Some comments on what these methods do needs to added or the method name made more descriptive.

Reply:
I agree